slice/Glacier2/Session.ice

```diff
@@ -5,135 +5,128 @@
+        /// @param deletions The sequence of Ice identities to remove.
         idempotent void remove(Ice::IdentitySeq deletions);
 
-        /// Returns a sequence of identities describing the constraints in this set.
+        /// Gets a sequence of identities describing the constraints in this set.
         /// @return The sequence of Ice identities for this set.
         ["swift:identifier:`get`"]
         idempotent Ice::IdentitySeq get();
     }
 
-    /// An administrative session control object, which is tied to the lifecycle of a {@link Session}.
-    /// @see Session
+    /// Represents a router-provided object that allows an application-provided session manager to configure the
+    /// routing constraints for a session.
     interface SessionControl
     {
-        /// Access the object that manages the allowable categories for object identities for this session.
-        /// @return A StringSet object. The returned proxy is never null.
+        /// Gets a proxy to the object that manages the allowable categories for object identities for this session.
+        /// @return A proxy to a StringSet object. This proxy is never null.
         StringSet* categories();
 
-        /// Access the object that manages the allowable adapter identities for objects for this session.
-        /// @return A StringSet object. The returned proxy is never null.
+        /// Gets a proxy to the object that manages the allowable adapter identities for objects for this session.
+        /// @return A proxy to StringSet object. This proxy is never null.
         StringSet* adapterIds();
 
-        /// Access the object that manages the allowable object identities for this session.
-        /// @return An IdentitySet object. The returned proxy is never null.
+        /// Gets a proxy to the object that manages the allowable object identities for this session.
+        /// @return A proxy to an IdentitySet object. This proxy is never null.
         IdentitySet* identities();
 
-        /// Get the session timeout.
+        /// Gets the session timeout.
         /// @return The timeout.
         idempotent int getSessionTimeout();
 
-        /// Destroy the associated session.
+        /// Destroys the associated session.
         void destroy();
     }
 
-    /// The session manager for username/password authenticated users that is responsible for managing {@link Session}
-    /// objects. New session objects are created by the {@link Router} object calling on an application-provided
-    /// session manager. If no session manager is provided by the application, no client-visible sessions are passed to
-    /// the client.
-    /// @see Router
-    /// @see Session
+    /// Represents an application-provided factory for session objects. You can configure a Glacier2 router with your
+    /// own SessionManager implementation; this router will then return the sessions created by this session manager to
+    /// its clients.
```

Comment:
What about adding a link or see to `PermissionsVerifier`.

Reply:
I would find it confusing. The session manager does not interact with the permissions verifier.